src/webhooks.ts

```
@@ -0,0 +1,52 @@
+import { TeamMembers } from "./data/types";
+import axios from "axios";
+import AccessToken from "./classes/AccessToken";
+import { config } from "./config";
+import type { TeamMember } from "./data/types";
+
+const accessTokenUtil = new AccessToken();
+
+async function registerWebhook(topicUrl: string, member_id: string) {
+  const webhooksApiUrl = "https://api.twitch.tv/helix/webhooks/hub";
+
+  const accessTokenData = await accessTokenUtil.get();
+
+  if (accessTokenData) {
+    const data = {
+      "hub.callback": `${process.env.TWITCH_API_CALLBACK_URL}/${member_id}`,
+      "hub.mode": "subscribe",
+      "hub.topic": topicUrl,
+      "hub.lease_seconds": 84600,
+    };
+
+    try {
+      const response = await axios.post(webhooksApiUrl, data, {
+        headers: {
+          Authorization: `Bearer ${accessTokenData.accessToken}`,
+          "Client-Id": process.env.CLIENT_ID,
+          "Content-Type": "application/json",
+        },
+      });
+    } catch (err) {
+      console.error(err);
```

Why are we swallowing the exception?